### ISSUE TITLE
Tag where most recent data is old shows no chart or error (320) - minor

### ISSUE THREAD

**Issue by DG12** [ruuvi/com.ruuvi.station.ios#1119]
![image](https://user-images.githubusercontent.com/2365961/198843965-f59b2a6e-bb3e-4960-82b8-1762300a64aa.png)
![image](https://user-images.githubusercontent.com/2365961/198844070-7e6fba90-97a0-4947-8211-997dbe03f441.png)

**Comment by priyonto** [comment]
iOS app stores only the last 10 days of data. If the last measurement is older than 10 days there will be no chart. However, a message showing no data available will be presented instead.

**Comment by markoaamunkajo** [comment]
Can confirm this in 2 days

**Comment by markoaamunkajo** [comment]
@priyonto I finally see how this empty charts screen happened. I have disconnected my gateway for quite some time now and waited for 10 days to pass. Now when I made clean install of 1.3.1(323) and signed in I see empty charts screen if I visit one of the sensors that were using my gateway. However, if I visit another chart that has data the empty charts now appear.

I never saw message showing no data available.



https://user-images.githubusercontent.com/50437378/204174563-3f57c5aa-6175-4436-b752-93dae7886d93.mov

**Comment by markoaamunkajo** [comment]
@priyonto I finally see how this empty charts screen happened. I have disconnected my gateway for quite some time now and waited for 10 days to pass. Now when I made clean install of 1.3.1(323) and signed in I see empty charts screen if I visit one of the sensors that were using my gateway. However, if I visit another chart that has data the empty charts now appear.

I never saw message showing no data available.



https://user-images.githubusercontent.com/50437378/204174563-3f57c5aa-6175-4436-b752-93dae7886d93.mov
